app/adminPage.html

```diff
@@ -114,14 +114,15 @@ <h2>Add new test</h2>
                         <div id="inputSubmissionConfirmation" class="alert" role="alert"></div>
                     </div>
                 </form>
-                <div class="new-question-container">
-                    <h2>Add new question</h2>
+                <div class="modify-question-container">
+                    <h2>Modify questions</h2>
                     <h4 class="search-heading">Current Question Count: <span id="question-count"></span></h4>
                     <a class="btn primary-button new-question-button" href="addQuestion.html">Add new question</a>
+                    <a class="btn primary-button new-question-button" href="editTests.html">Edit questions</a>
```

Comment:
Should we refactor the file name 'editTests' to 'editQuestions' to make it more semantic?